ksql-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -166,7 +168,7 @@ private static void handleFailure(final RoutingContext routingContext) {
   private static AuthHandler basicAuthHandler(final Server server) {
     final AuthProvider authProvider = new JaasAuthProvider(server, server.getConfig());
     final String realm = server.getConfig().getString(ApiServerConfig.AUTHENTICATION_REALM_CONFIG);
-    return BasicAuthHandler.create(authProvider, realm);
+    return new BasicAuthHandler(authProvider, realm);
```

Comment:
suggestion: using a class name different to the Vertx default may avoid confusion here by making it clear this is _not_ the default one. e.g. `KsqlBasicAuthHandler`.

Reply:
Done.